### PR TITLE
Navbar Components

### PR DESCRIPTION
**Pull request by mxdi9i7**

### CODE REVIEW

File: src/components/Icons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  if (size) {
+    Object.assign(iconProps, {
+      style: {
+        ...iconProps.style,
+        fontSize: size
+      }
+    });
+  }
+
+  console.log(dot, badge);
```
Comment (wsh941102) [inline src/components/Icons/index.tsx:59]: ?

**Comment by z695245771** [comment]
image
image
font weight is different from original vent

**Comment by z695245771** [comment]
image
image

There is border-bottom in their original page

**Comment by wsh941102** [comment]
> image
> image
> font weight is different from original vent

I think so, a little bit thin for me

**Pull request merged**